chia/rpc/rpc_server.py

```diff
@@ -44,22 +44,23 @@ async def stop(self):
             await self.websocket.close()
 
     async def _state_changed(self, *args):
-        change = args[0]
         if self.websocket is None:
             return None
```

Comment:
This function returns `None` either way.
I'd do `if self.websocket is not None` and then the body, removing all the `return`'s (or ending with just one). 
I didn't simply to hold the diff minor.